### ISSUE TITLE
confirmdialog-2.0.5.jar  does not point to a valid jar for a Class-Path reference

### ISSUE THREAD

**Issue by ghost** [samie/Vaadin-ConfirmDialog#17]
I get a bunch of warnings below when using this with vaadin 7.1.15 and deploying on wildfly 8.0.0

23:38:51,271 WARN  [org.jboss.as.server.deployment](MSC service thread 1-9) JBAS015960: Class Path entry vaadin-shared-7.0.0.jar in /content/xxx-web.war/WEB-INF/lib/confirmdialog-2.0.5.jar  does not point to a valid jar for a Class-Path reference.
23:38:51,272 WARN  [org.jboss.as.server.deployment](MSC service thread 1-9) JBAS015960: Class Path entry vaadin-theme-compiler-7.0.0.jar in /content/xxx-web.war/WEB-INF/lib/confirmdialog-2.0.5.jar  does not point to a valid jar for a Class-Path reference.
23:38:51,272 WARN  [org.jboss.as.server.deployment](MSC service thread 1-9) JBAS015960: Class Path entry commons-logging-1.1.1.jar in /content/xxx-web.war/WEB-INF/lib/confirmdialog-2.0.5.jar  does not point to a valid jar for a Class-Path refere

**Comment by dajoropo** [comment]
Hi Jon,

I'm having a similar problem with vaadin 7.5.0 and confirm dialog 2.1.3. Did you ever found a solution for these warnings?

Cheers

**Comment by jonmartinsolaas** [comment]
Hi, I’m on Vildfly 9.0.2 and Vaadin 7.6.1 now, getting the warnings on nearly all jars in the project. Everything seems to play nicely, though, so I consider them harmless.

**Comment by dajoropo** [comment]
Yes I thought it would be difficult to remove them... In my case the application also works perfectly, so I will ignore them also.

Many thanks for answering!!

**Comment by jonmartinsolaas** [comment]
If you use several vaadin add ons it is unlikely there will ever exist a configuration where all depend om the same vaadin-version ... :o)

**Comment by parameshjava** [comment]
I would like to use logback implementation rather than log4j implementation for my application. I am also facing this problem.
